assets/lokomotive-kubernetes/bootkube/variables.tf

```diff
@@ -139,3 +139,9 @@ variable "external_apiserver_port" {
   type        = number
   default     = 6443
 }
+
+variable "expose_on_all_ports" {
+  description = "Controls if kube-apiserver should listen on all interfaces on port 6443"
```

Comment:
If false, it which interface will it listen? Looking at the variables file, I don't see a clear answer. So, this seems confusing, at least for me as a user :)

Reply:
I meant `NodeIP` of type `InternalIP` (https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#nodeaddress-v1-core). Which you can see when running `kubectl get nodes -o wide`.

>what if the node has several private IPs? Which one or how it will pick it up?

I don't know how selection works then. We use `status.podIP` with `hostNetwork: true`. I guess that can be enforced by using `--node-ip` `kubelet flag.

Reply:
Added a comment to the code for it.

Reply:
Awesome, thanks!